### DISCUSSION TITLE
Node-sass 6 support

### DISCUSSION THREAD

**Discussion by lavcraft** [vercel/next.js#25844] [Ideas]
### Describe the feature you'd like to request

Hello everyone. Im trying to use next.js with existed project on node 16. But node-sass 4/5 doesn't work with node 16.

In this case a has a conflict with next.js peer dependencies (node-sass 4 || 5) and node 16 support for old node-sass versions (it doesn't compile, fail when npm rebuild node-sass)

Native node-sass 6 support can be a good choice for node 16 users

### Describe the solution you'd like

Support node-sass (may be need only valid peer dependency, i don't know)

### Describe alternatives you've considered

I can install with `--force` flag, but error about node-sass incompatible version happened when open page after `npm run dev`

**Comment by bnn1** [comment]
It has nothing to do with Next.js. It's just `node-sass` doesn't work with latest `node` yet.

**Reply by buesing** [reply]
That is not true. Next.js has defined a peer dependency:
```
    "node-sass": "^4.0.0 || ^5.0.0",
```
and this should be updated.

**Comment by lavcraft** [comment]
@bnn1 it's not right i guess. Because node-sass 6 support node 16 (see https://github.com/sass/node-sass/issues/3077)
next.js team should change peerDependency to allow node-sass 6 and fix related problems with node-sass

**Comment by bcspragu** [comment]
Just ran into the same issue. When using Node 16 and Next 10, one cannot `npm install sass`, as described [in the documentation](https://nextjs.org/docs/basic-features/built-in-css-support#sass-support). If there's no incompatibilities between Next 10 and Sass 6, the peerDependency should indeed be updated.

**Comment by evantahler** [comment]
I believe that there's a dependency chain on the newest version of `sass-loader`, and then `webpack` 5 to enable this

**Comment by ngobach** [comment]
`node-sass@6` seems to be not supported as now.
The current dependency of `sass-loader` link to version [10.0.5](https://github.com/vercel/next.js/blob/master/packages/next/package.json#L233) which doesn't support `node-sass@6`

**Comment by tihuan** [comment]
A PR has been merged to address this! https://github.com/vercel/next.js/pull/28710

Relevant discussion: https://github.com/vercel/next.js/discussions/28704
